packages/patternfly-4/react-integration/cypress/integration/radio.spec.ts

```diff
@@ -0,0 +1,26 @@
+describe('Radio Demo Test', () => {
+  it('Navigate to demo section', () => {
```

Comment:
Can you add some interactive test for checking and unchecking the controlled and uncontrolled buttons?

Reply:
It has one for the controlled button, but the uncontrolled buttons don't change at all HTML-wise when they're selected, so I don't know that you can write that kind of test for them.